Transformation: Added variance and standard deviation (#48844)
grafana/grafana: docs/sources/panels/calculation-types.md

> Refer to the following list of calculations you can perform in Grafana. You can find these calculations in the **Transform** tab and in the bar gauge, gauge, and stat visualizations.

Reviewer: ```suggestion
You can perform the following calculations in Grafana. Navigate to the **Transform** tab and in the bar gauge, gauge, and stat visualizations.
```
